mkdocs.yml

```diff
@@ -286,8 +241,51 @@ nav:
              - cloud/dbaas/databases.md
              - PostgreSQL: cloud/dbaas/postgresql.md
         - More advanced features: cloud/dbaas/advanced.md
-     - Grid computing:
-        - The Grid Monitor: cloud/grid/grid-monitor.md
```

Comment:
Making sure that everybody that reads this is aware: this document have been removed.

Reply:
Yup 🫣